poly-commitment/src/tests/batch_15_wires.rs

```diff
@@ -114,7 +114,34 @@ where
             );
             open += start.elapsed();
 
-            (sponge.clone(), x, polymask, evalmask, comm, proof)
+            let combined_inner_product = {
+                let es: Vec<_> = comm
+                    .iter()
+                    .map(|(commitment, evaluations, degree_bound)| {
+                        let bound: Option<usize> = (|| {
+                            let b = (*degree_bound)?;
+                            let x = commitment.commitment.shifted?;
```

Comment:
Is this double `commitment.commitment.shifted?` a typo? In `commitment.rs` this is done with just one level `commitment.shifted?`.

Reply:
Indeed: this is a blinded commitment, so we have to cast it to an unblinded one first